pydantic/main.py

```diff
@@ -100,17 +100,30 @@ def _extract_validators(namespace):
     return validators
 
 
+def inherit_validators(base_validators, validators):
+    for field, field_validators in base_validators.items():
+        if field in validators:
```

Comment:
if you want to do it this way

```py
if field not in validators:
    validators[field] = []
validators[field] += field_validators
```

Reply:
This is to avoid mutating the validators in the base classes.
```python
a = []
b = a
b += [1]
assert a == [1]
```

Reply:
what I've suggested has exactly the same effect as what you have now, just avoids duplicated lines.